autoload/OmniSharp/actions/usages.vim

```diff
@@ -38,10 +38,12 @@ endfunction
 
 function! s:CBFindUsages(target, opts, locations) abort
   let numUsages = len(a:locations)
-  if numUsages > 0
-    call OmniSharp#locations#SetQuickfix(a:locations, 'Usages: ' . a:target)
+  if numUsages == 0
+    echom 'No usages found'
```

Comment:
I don't think we need this echoed to `:messages`. Just `echo` is better than `echom` in this case.